Docs/Book/GettingStartedInPython.rst

```diff
@@ -64,10 +64,18 @@ or None on failure:
 
 .. doctest::
 
-  >>> m = Chem.MolFromMolFile('data/invalid.mol')
-  >>> m is None
+  >>> m_invalid = Chem.MolFromMolFile('data/invalid.mol')
+  >>> m_invalid is None
   True
 
+An :py:class:`rdkit.Chem.rdchem.Mol` object can be displayed graphically using :py:func:`rdkit.Chem.Draw.MolToImage`:
+
+.. doctest::
+
+  >>> Draw.MolToImage(m)
```

Comment:
Since the return value of `MolToImage` is an Image object, it makes sense to assign it to something:
```suggestion
.. doctest::

  >>> img = Draw.MolToImage(m)
```
This will also have the side effect of making the doctests pass.

Reply:
I implemented that change. Thanks.